src/TimeSliceGroup.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { PropTypes, Component } from 'react'
+import TimeSlice from './TimeSlice.jsx'
+import date from './utils/dates.js'
+
+export default class TimeSliceGroup extends Component {
+  static propTypes = {
+    slices: PropTypes.oneOf([2,3,4,5]),
+    size: PropTypes.number.isRequired,
+    format: PropTypes.string.isRequired,
+    value: PropTypes.instanceOf(Date).isRequired,
+    showlabels: PropTypes.bool,
+    isNow: PropTypes.bool,
+    timesliceClassnames: PropTypes.string,
+    debug: PropTypes.bool
+  }
+  static defaultProps = {
+    slices: 2,
+    isNow: false,
+    showlabels: true,
+    timesliceComponent: TimeSlice,
+    debug: false
+  }
+
+  renderSlice(i, className, value) {
+    return <TimeSlice key={i} showlabel={this.props.showlabels && !i} format={this.props.format}
+              isNow={this.props.isNow} classNames={className} value={value} />
+  }
+
+  renderSlices() {
+    let className = this.props.timesliceClassnames ? `${this.props.timesliceClassnames} ` : ''
+    switch (this.props.slices) {
```

Comment:
this isn't ideal...first the classes aren't prefixed, and they are limited arbitrarily. What is the value of the className here anyway? A user could use `nth-child` to target the one they want if need be.